code/obj/medicaldiagnosis.dm

```diff
@@ -0,0 +1,91 @@
+								playsound(user, "sound/effects/distortedfasthyperstethoscope.ogg", 40, 0, -6)
+							else if(M.organHolder && ((M.organHolder.left_lung && M.organHolder.left_lung.get_damage() > 50) || (M.organHolder.right_lung && M.organHolder.right_lung.get_damage() > 50)))
+								boutput(user, "<span style=\"color:red\">You hear coarse crackling noises inside [(user != M) ? "[M]'s" : "your"] lungs.</span>")
+								playsound(user, "sound/effects/cracklesstethoscope.ogg", 40, 0, -6)
+							else if(M.reagents.has_reagent("histamine")) // Are they poisoned with stuff that causes swelling (ADD MORE THINGS YOU NERD)
+								boutput(user, "<span style=\"color:red\">You hear what sounds like low-pitched wheezing inside [(user != M) ? "[M]'s" : "your"] lungs.</span>")
+								playsound(user, "sound/effects/stridorstethoscope.ogg", 40, 0, -6)
+							else if(M.reagents.has_reagent("krokodil") || M.reagents.has_reagent("morphine") || M.reagents.has_reagent("haloperidol") || M.reagents.has_reagent("ether") || (M.reagents.has_reagent("ethanol") && M.reagents.get_reagent_amount("ethanol") > 25))
+								boutput(user, "<span style=\"color:blue\">You hear very slow, shallow breathing inside [(user != M) ? "[M]'s" : "your"] lungs.</span>")
+								playsound(user, "sound/effects/sleepstethoscope.ogg", 40, 0, -6) // reusing because the sounds are similar
+							else if(M.reagents.has_reagent("cannabidiol") || M.reagents.has_reagent("antihistamine") || (M.reagents.has_reagent("ethanol") && M.reagents.get_reagent_amount("ethanol") < 25))
+								boutput(user, "<span style=\"color:blue\">You hear slightly slow breathing inside [(user != M) ? "[M]'s" : "your"] lungs.</span>")
+								playsound(user, "sound/effects/sleepstethoscope.ogg", 40, 0, -6) // reusing because the sounds are similar
+							else if(M.reagents.has_reagent("epinephrine") || M.reagents.has_reagent("methamphetamine") || M.reagents.has_reagent("crank") || M.reagents.has_reagent("stimulants") || M.reagents.has_reagent("energydrink") )
+								boutput(user, "<span style=\"color:blue\">You hear fast breathing inside [(user != M) ? "[M]'s" : "your"] lungs.</span>")
+								playsound(user, "sound/effects/hyperventstethoscope.ogg", 40, 0, -6) // reusing because the sounds are similar
+							else if((20 < M.get_oxygen_deprivation()) && (M.get_oxygen_deprivation() < 80)) // pick from 3 stressed breath sounds when I feel like adding them
+								boutput(user, "<span style=\"color:blue\">You hear what sounds like hyperventilation [M.get_oxygen_deprivation() > 60 ? "with an irregular pattern" : ""] inside [(user != M) ? "[M]'s" : "your"] lungs.</span>") // oxygen low but they can still recover
+								if(M.get_oxygen_deprivation() < 60)
+									playsound(user, "sound/effects/hyperventstethoscope.ogg", 40, 0, -6)
```

Comment:
these sounds should be added to the sound cache

Reply:
Added, this should be resolved.